forager/Env.py

```diff
@@ -112,8 +112,10 @@ def _respawn(self):
 
     def _get_observation(self, s: Coords):
         if self._c.observation_mode == 'objects':
+            assert self._ap_size is not None, 'Aperture size is None when observation mode is WORLD'
```

Comment:
Ideally this error message should match the conditions by which the user got here. By getting to this failure, the user has said:
```python
ForagerConfig(
  observation_mode='objects',
  aperture=None,
)
```
Likely, the user is wanting 'object'-mode observations and so would be confused why the error message is talking about 'world'-mode observations.

Maybe an alternative message could be:
```
'Expected non-none aperture size when observation mode is "objects"'
```